.github/workflows/main.yaml

```diff
@@ -0,0 +1,117 @@
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.go-version }}
+      - name: Set GCP instance name
+        run: echo "INSTANCE_NAME=etcdpasswd-${{ github.run_number }}-$(TZ=Asia/Tokyo date +%H%M%S)" >> $GITHUB_ENV
+      - run: |
+          if [[ ${{ github.event_name }} = "pull_request" ]]; then
+            GITHUB_SHA=$(cat ${GITHUB_EVENT_PATH} | jq -r '.pull_request.head.sha')
+          fi
+          ./bin/run-mtest.sh
+      - name: Set the instance lifetime
+        if: ${{ failure() ||  cancelled() }}
+        run: |
+          . ./bin/env
+          $GCLOUD compute instances add-metadata ${INSTANCE_NAME} --zone ${ZONE} \
+            --metadata shutdown-at=$(date -Iseconds -d+30minutes)
+      - name: Notify to Slack
+        if: ${{ failure() ||  cancelled() }}
+        run: |
+          . ./bin/env
+          if [[ ${{ github.event_name }} == 'pull_request' ]]; then
+              BRANCH_NAME=${GITHUB_HEAD_REF}
+          else
+              BRANCH_NAME=${GITHUB_REF#refs/heads/}
+          fi
+          curl -X POST -H 'Content-type: application/json' --data "{
+            \"blocks\": [
+              {
+                \"type\": \"section\",
+                \"text\": {
+                  \"type\": \"mrkdwn\",
+                  \"text\": \"Failed: ${{ github.actor }}'s workflow (${{ github.workflow }}) in <https://github.com/${{ github.repository }}/actions/workflows/${{ env.filename }}|${{ github.repository }}> (<https://github.com/${{ github.repository }}/actions/workflows/${{ env.filename }}?query=branch%3A${BRANCH_NAME}|${BRANCH_NAME}>)\n Do you extend the lifetime of ${INSTANCE_NAME}?\"
+                },
+                \"accessory\": {
+                  \"type\": \"button\",
+                  \"text\": {
+                    \"type\": \"plain_text\",
+                    \"text\": \"Extend\",
+                    \"emoji\": true
+                  },
+                  \"value\": \"${INSTANCE_NAME}\"
+                }
+              }
+            ]
+          }" ${{ secrets.SLACK_WEBHOOK_URL }}
+  release:
+    name: Release
+    needs: mtest
+    runs-on: ubuntu-20.04
+    if: startsWith(github.ref, 'refs/tags/v')
+    container:
+      image: 1.16.2.1_focal
+    steps:
+      - run: mkdir -p /tmp/upload
+      - name: Download etcdpasswd deb package
+        uses: actions/download-artifact@v2
+        with:
+          name: etcdpasswd
+          path: /tmp/upload
+      - run: |
+          TAG=${GITHUB_REF#refs/tags/v}
+          OWNER=$(echo ${{ github.repository }} | cut -d '/' -f 1)
+          REPOSITORY=$(echo ${{ github.repository }} | cut -d '/' -f 2)
+          if ${{ contains(github.ref, '-') }}; then
+            PRERELEASE="-prerelease"
+          fi
+          ghr -t ${{ secrets.GITHUB_TOKEN }} -u ${OWNER} -r ${REPOSITORY} -n ${TAG} ${PRERELEASE} -b "See [CHANGELOG.md](./CHANGELOG.md) for details." ${TAG} /tmp/upload
```

Comment:
The URL of a release page is as follows.
https://github.com/cybozu-go/etcdpasswd/releases/tag/v1.1.0

So this link will open the following link, I think.
https://github.com/cybozu-go/etcdpasswd/releases/tag/v1.1.0/CHANGELOG.md
(or https://github.com/cybozu-go/etcdpasswd/releases/tag/CHANGELOG.md)

But the actual URL for CHANGELOG.md is below.
https://github.com/cybozu-go/etcdpasswd/blob/v1.1.0/CHANGELOG.md

Reply:
I think this is fine.
See [the releases in coil](https://github.com/cybozu-go/coil/releases)